src/main/java/domain/game/ChessGame.java

```diff
@@ -0,0 +1,54 @@
+package domain.game;
+
+import domain.piece.Color;
+import domain.piece.Piece;
+import domain.position.Position;
+import java.util.Map;
+
+public class ChessGame implements Execute {
+    private final ChessBoard chessBoard;
+    private Color color;
+    private GameState gameState;
+
+    public ChessGame() {
+        this.chessBoard = new ChessBoard();
+        this.color = Color.WHITE;
+        this.gameState = GameState.READY;
+    }
+
+    public boolean isNotEnd() {
+        return gameState != GameState.END;
+    }
+
+    public Map<Position, Piece> getChessBoard() {
+        return chessBoard.getPieceByPosition();
+    }
+
+    @Override
+    public void start() {
+        if (gameState == GameState.RUNNING) {
```

Comment:
```suggestion
        if (gameState.isRunning()) {
```
과 같은 메서드를 통해 객체에게 물어볼까요?

Reply:
[반영완료]
State에게 메시지를 보내는 형식으로 수정했습니다.